crates/sui-json-rpc/src/api.rs

```diff
@@ -342,80 +343,82 @@ pub trait EventStreamingApi {
 #[open_rpc(namespace = "sui", tag = "Event Read API")]
 #[rpc(server, client, namespace = "sui")]
 pub trait EventReadApi {
-    /// Return list of events emitted by a specified transaction.
+    /// Return events emitted by the given transaction.
     #[method(name = "getEventsByTransaction")]
     async fn get_events_by_transaction(
         &self,
         /// digest of the transaction, as base-64 encoded string
         digest: TransactionDigest,
+        /// maximum size of the result, capped to 100
+        count: usize,
     ) -> RpcResult<Vec<SuiEventEnvelope>>;
 
-    /// Return list of events emitted by a specified Move module
+    /// Return events emitted in a specified Move module
     #[method(name = "getEventsByModule")]
-    async fn get_events_by_module(
+    async fn get_events_by_transaction_module(
         &self,
         /// the Move package ID
         package: ObjectID,
         /// the module name
         module: String,
-        /// maximum size of the result
-        count: u64,
+        /// maximum size of the result, capped to 100
```

Comment:
is this always 100? Will this change by config?

Reply:
It is hardcoded to be 100 per https://github.com/MystenLabs/sui/blob/fbebfb8abbbb89cedd8bb34943f99f7e9d66056d/crates/sui-storage/src/event_store/mod.rs#L39 
I'm not sure if we want to expose this in config, wdyt? 
It's indeed hard to maintain the doc with this number everywhere though. I will export the value to this file and refer to it while we are thinking about it